<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wf_plot</class>
 <widget class="QWidget" name="wf_plot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1190</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCustomPlot" name="wf_graphQCustom" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>841</width>
     <height>481</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="close_plot_Button">
   <property name="geometry">
    <rect>
     <x>1070</x>
     <y>460</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>CLOSE</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>10</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Energy_mEv</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="selEnergyLineEdit">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>40</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="K_barPlotText">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>290</y>
     <width>141</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="K_wellPlotText">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>290</y>
     <width>141</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>250</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>K_bar</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>250</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>K_well</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>170</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>std_deviation_nm</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="deviat_lineEdit">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>200</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>80</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Average_nm</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="average_lineEdit">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>110</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>Plot/qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>close_plot_Button</sender>
   <signal>clicked()</signal>
   <receiver>wf_plot</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>461</y>
    </hint>
    <hint type="destinationlabel">
     <x>133</x>
     <y>498</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
